src/main/java/pro/sky/telegrambotanimalshelter/controller/HumanDogController.java

```diff
@@ -0,0 +1,115 @@
+package pro.sky.telegrambotanimalshelter.controller;
+
+
+import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.Parameter;
+import io.swagger.v3.oas.annotations.media.Content;
+import io.swagger.v3.oas.annotations.media.Schema;
+import io.swagger.v3.oas.annotations.responses.ApiResponse;
+import org.springframework.http.MediaType;
+import org.springframework.web.bind.annotation.*;
+import pro.sky.telegrambotanimalshelter.models.HumanDog;
+import pro.sky.telegrambotanimalshelter.service.implementation.HumanDogServiceImpl;
+
+import java.util.Collection;
+
+
+@RestController
+@RequestMapping("human-dog")
+public class HumanDogController {
+
+    private final HumanDogServiceImpl personDogService;
+
+    public HumanDogController(HumanDogServiceImpl personDogService) {
```

Comment:
```suggestion
    public HumanDogController(HumanDogService personDogService) {
```
зависеть надо от абстракций, а не реализаций